src/desktop/pch.h

```diff
@@ -0,0 +1,22 @@
+// PCH list based on ClangBuildAnalyzer output
```

Comment:
Clang has a `-ftime-trace` flag that makes it output a `chrome://tracing` compatible `.json` file. `ClangBuildAnalyzer` reads all the `.json` file in a directory and outputs a human readable summary. Not sure if it could be parsed automatically or not. Nor if it could resolves path automatically (ClangBuildAnalyzer only shows the absolute paths).